020-quarkus-http-non-application-endpoints/src/test/java/io/quarkus/qe/non_application/endpoint/BackwardCompatibilityTest.java

```diff
@@ -10,13 +10,15 @@
 
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
 import io.quarkus.qe.http.non_application.endpoint.HelloResource;
 import io.quarkus.test.QuarkusProdModeTest;
 
+@Disabled("2.x is no longer backward compatible and hence not redirecting endpoints")
```

Comment:
@pjgg if this test is not longer valid, should we delete it? wdyt?

Reply:
Yes, I think that should be removed

Reply:
Deleted